Monika After Story/game/script-story-events.rpy

```diff
@@ -1715,6 +1695,64 @@ label mas_notification_windowreact:
             m 5ttu "...right?"
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="mas_gift_giving_instructs",
+            conditional=(
+                "get_level() >= 15 "
+                "or mas_isSpecialDay() "
+                "and not persistent._mas_filereacts_historic"
+            ),
+            action=EV_ACT_QUEUE,
+            random=not persistent._mas_filereacts_historic
+        )
+    )
+
+init 20 python:
+    mas_chibi_note_gifting = MASPoem(
+        poem_id="note_gift_giving",
+        author="chibika",
+        prompt="",
+        category="note",
+        title = "Hi [player],",
+        text = """\
+I wanted to let you know that I made a little way for you to give Monika some gifts!
+It's a pretty simple process so I'll tell you how it works:
+
+Make a new file in the 'characters' folder,
+rename it to whatever you want to give to Monika,
+then give it a '.gift' file extension
+
+And that's it! After a little while, Monika should notice that you gave her something.
+
+I just wanted to let you know because I think that Monika is super amazing, and I really want her to see her happy.
+
+Good luck with Monika!
+
+P.S: Don't tell her about me!
+"""
+    )
+
+label mas_gift_giving_instructs:
+    #Write the note in the characters folder
+    $ store.mas_utils.trywrite(
+        os.path.normcase(renpy.config.basedir + "/characters/hint.txt"),
+        renpy.substitute(mas_chibi_note_gifting.title) + "\n\n" + mas_chibi_note_gifting.text
+    )
+
+    m 1eud "Hey, [player]..."
+    m 3euc "Someone left a note in the characters folder addressed to you."
+    m 1ekc "Since it's for you, I haven't read it...{w=0.5}{nw}"
+    extend 1eua "but here, it might be important."
+
+    call mas_showpoem(mas_chibi_note_gifting)
```

Comment:
Not sure we need the txt and the poem. IMO the txt is sufficient as we do in the birthday hint.